.vscode/settings.json

```diff
@@ -20,7 +20,6 @@
   // Standardize on the Prettier formatter
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   // Enforce prettier on save
-  "editor.formatOnSave": true,
```

Comment:
Curious why we're removing Prettier from here and the recommended extension list?